lib/kb_kaiju/kb_kaijuImpl.py

```diff
@@ -46,9 +46,9 @@ class kb_kaiju:
     # state. A method could easily clobber the state set by another while
     # the latter method is running.
     ######################################### noqa
-    VERSION = "1.2.0"
+    VERSION = "1.2.1"
```

Comment:
You have "1.3.0" in the release notes - swap "1.2.1" for the "1.3.0" here

Reply:
done